### PR TITLE
Adding new Clickhouse fields for NP visibility

### PR DESCRIPTION
**Pull request by tushartathgur**
Introduced new fields as per go-ipfix changes for NP visibility

### CODE REVIEW

**Review by Dhruv-J — APPROVED** [review]
LGTM

File: build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql [hunk 1]
```diff
@@ -212,3 +212,9 @@ ALTER TABLE tadetector_local
     DROP COLUMN aggType,
     DROP COLUMN direction,
     DROP COLUMN podName;
+ALTER TABLE flows
```
Comment (yuntanghsu) [inline build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql:215]: Should these changes be in this file? Or it should be 000006_0-7-0?

File: build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.up.sql [hunk 2]
```diff
@@ -151,3 +151,9 @@ ALTER TABLE tadetector_local
     ADD COLUMN aggType String,
     ADD COLUMN direction String,
     ADD COLUMN podName String;
+ALTER TABLE flows
```
Comment (yuntanghsu) [inline build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.up.sql:154]: ditto

File: build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql (hunk 1 above)
Comment (tushartathgur) [reply build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql]: Changed

File: build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.up.sql (hunk 2 above)
Comment (tushartathgur) [reply build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.up.sql]: changed!

**Review by yuntanghsu — APPROVED** [review]

**Pull request closed without merging**